test/form/index.js

```diff
@@ -1,12 +1,13 @@
 const path = require('path');
+const fs = require('fs');
 const assert = require('assert');
 const sander = require('sander');
 const rollup = require('../../dist/rollup');
 const { extend, loadConfig, normaliseOutput } = require('../utils.js');
 
 const samples = path.resolve(__dirname, 'samples');
 
-const FORMATS = ['amd', 'cjs', 'es', 'iife', 'umd'];
+const FORMATS = ['amd', 'cjs', 'es', 'iife', 'umd', 'system'];
```

Comment:
Let me know your thoughts on this -- the vast majority of the tests do not have an `_expected/system.js` file defined for the "form" tests. But I'm thinking that all future tests should.

Reply:
This is included in the PR at https://github.com/rollup/rollup/pull/2022. Perhaps rebase to that if you want to do this?